builtin/mainmenu/tab_about.lua

```diff
@@ -40,21 +40,17 @@ local core_team = {
 
 local active_contributors = {
 	"Wuzzy [Features, translations, devtest]",
-	"Lars Müller [Lua optimizations and fixes]",
-	"Jude Melton-Houghton [Optimizations, bugfixes]",
-	"paradust7 [Performance, fixes, Irrlicht refactoring]",
-	"Desour [Fixes]",
-	"ROllerozxa [Main menu]",
-	"savilli [Bugfixes]",
+	"Lars Müller [Bugfixes and entity features]",
+	"paradust7 [Bugfixes]",
+	"ROllerozxa [Bugfixes, Android]",
+	"srifqi [Android, translations]",
 	"Lexi Hale [Particlespawner animation]",
-	"Liso [Shadow Mapping]",
-	"JosiahWI [Fixes, build system]",
-	"numzero [Graphics and rendering]",
-	"HybridDog [Fixes]",
-	"NeroBurner [Joystick]",
-	"pecksin [Clickable web links]",
+	"savilli [Bugfixes]",
+	"fluxionary [Bugfixes]",
+	"Gregor Parzefall [Documentation]",
```

Comment:
Can `[Documentation]` be changed into `[Bugfixes]` or something? That's what I did the most according to https://github.com/minetest/minetest/pulls?q=is%3Apr+author%3Agrorp+is%3Amerged . I made very few changes changes to the documentation.